pkg/operator/starter.go

```diff
@@ -305,6 +305,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder.WithComponentSuffix("invalid-webhook-certs"),
 		"/var/run/configmaps/service-ca-bundle/service-ca.crt",
 		metricscontroller.NewLegacyCNCertsMetricsSyncFunc(
+			"",
```

Comment:
@s-urbaniak you didn't leave instructions on how to update after the changes introduced in https://github.com/openshift/library-go/pull/1289 :). Is it okay to pass an empty prefix?

Reply:
@p0lyn0mial my bad, i was expecting to do this bump myself :-)
Two comments:
1. Yes, it can be empty string. In this case, conditions will not be prefixed for status reporting. This is only important if you have multiple metrics controllers in the same operator or you want to give more context. here, empty string is fine.
2. we're a bit smarter now with the detection what certificates are faulty. For this the query can return labels to differentiate. I gave a suggestion below.